trunk/workflow-manager/src/test/java/org/mitre/mpf/wfm/camelOps/TestTrackMergingProcessor.java

```diff
@@ -90,77 +90,41 @@ public int next() {
 
```

Comment:
This line can be removed:

`private static final MutableInt SEQUENCE = new MutableInt();`

So can the `next()` method.